src/kvstore/Listener.h

```diff
@@ -77,15 +77,17 @@ using RaftClient = thrift::ThriftClientManager<raftex::cpp2::RaftexServiceAsyncC
  *   // extra initialize work could do here
  *   void init()
  *
+ *   // Main interface to process logs, listener need to apply the committed log entry to their
+ *   // state machine. Once apply succeeded, user should call persist() to make their progress
+ *   // persisted.
+ *   virtual void processLogs() = 0;
```

Comment:
I have a small question:
Why does the function comment here exist? Can it be removed?
Otherwise, there are two copies of these comments, which feels a bit redundant?